### ISSUE TITLE
How will roapi handle the annotations

### ISSUE THREAD

**Issue by sckott** [ropensci/roapi#1]
They'll be a table we manually update with those kind of notes, called `repos`, SQL create code here https://github.com/ropensci/roapi/blob/master/sqlnotes.txt#L1-L16

**Comment by sckott** [comment]
ping @cboettig

**Comment by cboettig** [comment]
nice :+1:

Could we have a way to denote when the vignette cannot be built (or rather, isn't trivial to build..) e.g. RRedisAPI needs a redis server actually running to build the vignette (at least it does at the moment, I think @richfitz might have changed that to use a dummy vignette in `vignettes` to keep CRAN happy).  

I just need some kind of flag that `drat` can use to turn on `--no-build-vignettes`, which would be preferable to having `--no-build-vignettes` be the default for everyone...

**Comment by sckott** [comment]
> Could we have a way to denote when the vignette cannot be built (or rather, isn't trivial to build..)

Yeah, maybe a field called `build_vignettes` as a boolean, true means can be built, and false can not

**Comment by cboettig** [comment]
:+1:
